pgmgr/pgmgr_test.go

```diff
@@ -257,6 +257,20 @@ func TestMigrate(t *testing.T) {
 
 	psqlMustExec(t, `SELECT * FROM bars;`)
 
+	// Make a filename that would match a vim .swp file
+	writeMigration(t, ".003_this_is_an_older_migration.up.sql.swp", `
+		CREATE TABLE baz (baz_id INTEGER);
+		INSERT INTO baz (baz_id) VALUES (4), (5), (6);
+	`)
+
+	err = Migrate(globalConfig())
+	if err != nil {
+		t.Log(err)
+		t.Fatal("Could not apply third migration!")
```

Comment:
Yeah, that would make sense since the expectation is for the migration not to occur.

Here is the change: afebd1bcf872c80358c62e494f6cbaad36b3f65b